be/src/exprs/jit/jit_engine.cpp

```diff
@@ -223,9 +219,13 @@ void JITEngine::print_module(const llvm::Module& module) {
     LOG(INFO) << "JIT: Generated IR:\n" << str;
 }
 
-void* JITEngine::lookup_function(const std::string& expr_name) {
+void* JITEngine::lookup_function(const std::string& expr_name, bool mast_exist) {
```

Comment:
must_exist?

Reply:
The main purpose here is to reduce the output of logs.

Reply:
mask_exist or must ? seems a typo